packages/doc-site/stories/data-sources/AWSIoTSiteWiseSource.mdx

```diff
@@ -160,6 +160,14 @@ Each asset contains the following fields:
 
   Type: String
 
+`requestSettings`
+
+(Optional) Specifies how the query requests time series data.
```

Comment:
(Optional) Specifies configuration settings for how the query requests time-series data.